dxr/plugins/core.py

```diff
@@ -429,6 +429,11 @@ class FileToIndex(dxr.indexers.FileToIndex):
     def __init__(self, path, contents, plugin_name, tree, vcs):
         super(FileToIndex, self).__init__(path, contents, plugin_name, tree)
         self.vcs = vcs
+        self.extension = splitext(self.path)[1].lower()
```

Comment:
Good points, for now I will add the case to the table. If I get the chance I'll file a bug on the usage of SVG instead of svg.